### PR TITLE
chore: Fix DpTab Storybook error.

### PR DESCRIPTION
**Pull request by ahmad-demos**
- Fix DpTab Storybook error.
`this.tabs[0] is undefined`

### CODE REVIEW

**Review by salisdemos — APPROVED** [review]

File: src/components/DpTabs/DpTabs.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/vue'
 import DpTabs from './'
-import DpTab from './'
+import DpTab from './DpTab.vue'
```
Comment (salisdemos) [inline src/components/DpTabs/DpTabs.stories.tsx:3]: why do we need a explicit import for DpTab but not for DpTabs

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpTabs/DpTabs.stories.tsx (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpTabs/DpTabs.stories.tsx:3]: DpTabs imported from index.ts
```
export { default } from './DpTabs.vue'
export * from './DpTabs.vue'
```

Comment (salisdemos) [reply src/components/DpTabs/DpTabs.stories.tsx:3]: ok. So we could do it for tab.vue the same way?

Comment (ahmad-demos) [reply src/components/DpTabs/DpTabs.stories.tsx:3]: there can only ever be one default export!

Comment (ahmad-demos) [reply src/components/DpTabs/DpTabs.stories.tsx:3]: @spiess-demos

Comment (spiess-demos) [reply src/components/DpTabs/DpTabs.stories.tsx:3]: as we need both components within the same story, but the `import x from ./` syntax only works for default exports, which we cannot have two at the same time within one file, the solution `import DpTab from './DpTab.vue'` is ok imho.

**Review by salisdemos — APPROVED** [review]

**Pull request merged**